code/modules/mob/living/carbon/resist.dm

```diff
@@ -2,7 +2,7 @@
 
 	//drop && roll
 	if(on_fire && !buckled)
-		fire_stacks -= 1.2
+		fire_stacks -= 2.5
```

Comment:
While you're at it, the weaken ends sooner than the spin animation.

Reply:
yea i noticed this needed buffed while testing flamethrowers